src/cmd2.cc

```diff
@@ -717,41 +657,6 @@ static void chest_death(int y, int x, s16b o_idx)
 
 
 /*
- * Chests have traps too.
```

Comment:
In my (hacked-up) games literally *every* chest shows up as "(empty)". I must admit I don't think I've actually tried to open them -- I just squelch all chests immediately.